src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -79,50 +83,113 @@ The API used to retrieve the value is:
   The number of times garbage collection has occurred for the specified generation
 of objects.
 
-#### Additional GC metrics only available when targeting .NET Core 3.1 or later
+#### process.runtime.dotnet.**gc.allocations.size**
+
+| Units   | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
```

Comment:
there is no attributes, so maybe make it clear that this metric has no dimensions?

Reply:
Updated to `| No Attributes    | N/A              |`.